### ISSUE TITLE
Accorder le titre du post et le titre de la ressource

### ISSUE THREAD

**Issue by balessan** [assemblee-virtuelle/wpldp#4]
Lorsqu'on édite ou crée une ressource, si on édite le champ post_name ou resource_name ou resource_title, que les deux soient coordonnés.

**Comment by sylvainlb** [comment]
Effectivement, c'est perturbant, surtout pour quelqu'un qui ne comprend pas qu'on tord wordpress

**Comment by balessan** [comment]
Tu as une idée de comment procéder ?
Il faut un titre de post, car c'est lui qui permet de générer l'URL.
Or on a presque forcément un title ou un name dans la ressource, mais sans garantie de pouvoir matcher le nom de l'attribut précisement.

Si on vire le ldp_ partout on peut peut-être faire correspondre les choses plus facilemement ?

**Comment by sylvainlb** [comment]
On pourrait prendre le title et l'ajouter systématiquement dans la ressource, voire le mettre par défaut dans les included fields.

Virer les ldp_, ça parait sain, mais comment tu fais pour pas te retrouver avec des champs wordpress dans toutes tes ressources ? Ou alors c'est un faux problème ?

**Comment by balessan** [comment]
Faut que je regarde, mais je crois que les champs associés au modèle sont stockés sous forme de meta, pas sous forme de fields du champs. Si c'est le cas il n'y aura pas de conflit.

À tester.
